sources/org.osbuild.ostree

```diff
@@ -30,13 +31,22 @@ def main(options, checksums, cache, output):
         remote = commits[commit]["remote"]
         url = remote["url"]
         uid = remote["uid"]
+        gpg = remote.get("gpgkeys", [])
+
+        extra_args = []
```

Comment:
In the `org.osbuild.rpm` stage we do the gpg verification in the stage, rather than in the source. We should probably unify here. I was thinking the gpg keys would be a property of the given manifest, and that they should be rechecked if the next manifest uses different keys, but the data is still available in the source cache, but maybe it makes sense to verify on import?

Reply:
Good point. This somehow felt natural to me and I forgot to check what the rpm stage does. I felt that once it is in the cache it is trusted but maybe re-checking is also a good idea? I can argue both sides in my head and they both seem to make sense one way or the other.

Reply:
Yeah, I can see both sides too... Something to discuss (and document as a principle, I guess).